docs/MainSite/Installation.md

```diff
@@ -42,20 +42,67 @@ See LICENSE.txt for details.
 
 A [Docker](https://www.docker.com/) image is available from
 [DockerHub](https://hub.docker.com/r/sxscollaboration/spectrebuildenv/) and can
-be used to build SpECTRE on a personal machine. To retrieve the Docker image run
-
-```
-docker pull sxscollaboration/spectrebuildenv:latest
-```
-
-The image already has Charm++ configured, all that needs to be done is to
-clone SpECTRE and compile it.
+be used to build SpECTRE on a personal machine.
 
 **Note**: The Docker image is the recommended way of using SpECTRE on a personal
 Linux machine. Because of the wide variety of operating systems available today
 it is not possible for us to support all configurations. However, using Spack
 as outlined below is a supported alternative to Docker images.
 
+To build with the docker image:
+
+1. Clone spectre into SPECTRE_ROOT, a directory of your choice.
+2. Retrieve the docker image (you may need `sudo` in front of this command)
+   ```
+   docker pull sxscollaboration/spectrebuildenv:latest
+   ```
+3. Start the docker container (you may need `sudo`)
+   ```
+   docker run -v SPECTRE_ROOT:SPECTRE_ROOT -i -t sxscollaboration/spectrebuildenv:latest /bin/bash
```

Comment:
It could be useful to give the container a name by using `--name CONTAINER_NAME`. This name can be used to restart the container below